db/seeds.rb

```diff
@@ -0,0 +1,63 @@
+require 'faker'
+
+
+# Change these if you want
+DEFAULT_USERS = 10
+DEFAULT_TREES = (1..3).to_a
+DEFAULT_TREE_DEPTH = (2..4).to_a
+DEFAULT_BRANCHING_FACTOR = (2..4).to_a
+DEFAULT_VALUES = (-100..100).to_a
+
+num = ARGV[1].to_i
+num = DEFAULT_USERS if num == 0 # create 10 users by default
+
+def seed_tree(args = {})
```

Comment:
This method is way way too big. Break it up into small methods with descriptive names.